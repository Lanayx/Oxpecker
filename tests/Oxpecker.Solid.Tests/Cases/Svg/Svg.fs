module Oxpecker.Solid.Tests.Cases.Svg

open Oxpecker.Solid
open Oxpecker.Solid.Svg


[<SolidComponent>]
let SvgTest () =
    circle(``fill-opacity`` = "fillOpacity", ``clip-rule`` = "clipRule")
