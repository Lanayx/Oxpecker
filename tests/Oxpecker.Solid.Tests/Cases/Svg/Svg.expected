
export function SvgTest() {
    return <svg>
        <defs>
            <linearGradient id="gradient"
                x1="0%"
                y1="0%"
                x2="100%"
                y2="0%">
                <stop offset="0%"
                    style="stop-color:rgb(255,255,0);stop-opacity:1"></stop>
                <stop offset="100%"
                    style="stop-color:rgb(255,0,0);stop-opacity:1"></stop>
            </linearGradient>
        </defs>
        <circle id="circle"
            fill-opacity="50%"
            clip-rule="nonzero"></circle>
        Sorry but this browser does not support inline SVG.
    </svg>;
}

